enriched the readme (#14)
kyma-incubator/terraform-provider-gardener: README.md

## Overview

> The Gardener Terraform provider enables [Terraform](https://www.terraform.io) provisioning [Kubernetes](https://kubernetes.io) clusters on chosen cloud providers using [Gardener](https://gardener.cloud/). Currently, it supports AWS, Azure, and GCP.

Reviewer: ```suggestion
The Gardener Terraform provider enables [Terraform](https://www.terraform.io) to provision [Kubernetes](https://kubernetes.io) clusters on chosen cloud providers using [Gardener](https://gardener.cloud/). Currently, it supports AWS, Azure, and GCP.
```

## Prerequisites

- [Terraform](https://www.terraform.io/downloads.html) 0.10+
- [Go](https://golang.org/doc/install) 1.12 or higher

> - a Gardener project with kubeconfig access and configured cloud provider secrets

Reviewer: ```suggestion
-  Gardener project with kubeconfig access and configured cloud provider secrets
```

    ```bash
    go build -o terraform-provider-gardener
    ```
3. Move the gardener provider binary into the terraform plugins folder:

   On MacOS:
   ```bash
   mv terraform-provider-gardener ~/.terraform.d/plugins/terraform-provider-gardener
   ```

>    For other systems and further details see [this](https://www.terraform.io/docs/plugins/basics.html#installing-plugins) document.

Reviewer: ```suggestion
   For other systems and further details, see [this](https://www.terraform.io/docs/plugins/basics.html#installing-plugins) document.
```
